front-end/src/IndividualShow.js

```diff
@@ -76,22 +76,21 @@ const Description = ({ genre, description, totalEpisodes, isMovieN }) => {
   const refDescription = useRef();
   const refGenre = useRef();
   const refTotalEpisodes = useRef();
-  let movieV = { isMovieN };
-  if (isMovieN) {
-    movieV = "Yes";
-  } else {
-    movieV = "No";
-  }
 
   return (
     <div className="description">
       <br />
       <label className="descript" htmlFor="genre">Genre:  </label>
       <span className="descript" ref={refGenre}>{genre}</span> <br/>
       <label className="descript" htmlFor="description">Description:  </label>
-      <span className="descript" value={description} ref={refDescription}>{description} </span> <br/>
-      <label className="descript" htmlFor="totalEpisodes">Total Episodes: </label>
-      <span className="descript" ref={refTotalEpisodes}>{totalEpisodes}.</span>
+      <span className="descript" value={description} ref={refDescription}>{description} </span>
+      { !isMovieN
+        ? <React.Fragment>
+          <br/>
+          <label className="descript" htmlFor="totalEpisodes">Total Episodes: </label>
+          <span className="descript" ref={refTotalEpisodes}>{totalEpisodes}.</span>
+        </React.Fragment>
```

Comment:
I think instead of `<React.Fragment>` we've been using `<>` and `</>` which is a shorter syntax for it--would you mind changing this? https://reactjs.org/docs/fragments.html#short-syntax

Reply:
Changed